packages/ui/src/components/va-input-wrapper/VaInputWrapper.vue

```diff
@@ -2,9 +2,13 @@
   <div
     class="va-input-wrapper"
     :class="wrapperClass"
-    :style="wrapperStyle"
     @click="$emit('click', $event)"
   >
+    <VaInputLabel
```

Comment:
Issue:
<img width="468" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/23530004/0f36b093-490e-45d8-80d1-4061a5d122ac">

What to do with long label? Add ellipses?
<img width="486" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/23530004/5fe9e7ee-468e-4869-af31-75c1b0cde341">

Or label should make input bigger.
<img width="433" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/23530004/9b3ac51e-e2f2-4c00-bb1e-e6f3c74ff36f">